src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -706,6 +717,9 @@ private void CheckSize(int countToAdd)
             }
         }
 
+#if NET46
+        [SecuritySafeCritical]
```

Comment:
These don't exist in .Net Framework. Why are they present in the NET46 compilation? (Are these fixes or enhancements that we would like to preserve  in .Net Framework?)